appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -16,31 +16,47 @@ type ExtendedSkuDescription = WebSiteManagementModels.SkuDescription & { label?:
 
 export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWizardContext> {
     public async prompt(wizardContext: IAppServiceWizardContext): Promise<void> {
-        let skus: ExtendedSkuDescription[] = wizardContext.advancedCreation ? this.getRecommendedSkus().concat(this.getAdvancedSkus()) : this.getRecommendedSkus();
-        if (wizardContext.newSiteKind === AppKind.functionapp) {
-            skus.push(...this.getElasticPremiumSkus());
-        } else if (wizardContext.newSiteKind?.includes(AppKind.workflowapp)) {
-            skus = this.getWorkflowStandardSkus();
-        }
-
+        let recommendedSkus = this.getRecommendedSkus();
         const regExp: RegExp | undefined = wizardContext.planSkuFamilyFilter;
         if (regExp) {
-            skus = skus.filter(s => !s.family || regExp.test(s.family));
+            recommendedSkus = recommendedSkus.filter(s => !s.family || regExp.test(s.family));
         }
-
-        const pricingTiers: IAzureQuickPickItem<WebSiteManagementModels.SkuDescription | undefined>[] = skus.map(s => {
+        const pricingTiers: IAzureQuickPickItem<WebSiteManagementModels.SkuDescription | undefined>[] = recommendedSkus.map(s => {
             return {
                 label: s.label || nonNullProp(s, 'name'),
                 description: s.description || s.tier,
-                data: s
+                data: s,
+                group: localize('recommendedSkusLabel', 'Recommended')
             };
         });
 
+        let advancedSkus: ExtendedSkuDescription[] = [];
+        if (wizardContext.advancedCreation) {
+            advancedSkus.push(...this.getAdvancedSkus());
+            if (regExp) {
+                advancedSkus = advancedSkus.filter(s => !s.family || regExp.test(s.family));
+            }
+            if (wizardContext.newSiteKind === AppKind.functionapp) {
```

Comment:
Previously the elastic premium and workflow skus were being added regardless of `advancedCreation`, but I don't remember if that matters or not.

As a general note, you should make sure to test both App Service and Functions when modifying code like this that affects both. In particular, I'm curious if the sku prompt still looks right if I chose "Premium" for the hosting plan